spacetemplate/assets/legacy.yaml

```diff
@@ -219,6 +239,8 @@ work_item_type_groups:
   - *experienceID
   - *valuePropositionID
   bucket: portfolio
+  description: Work item type group that focuses on the experience and value
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed.